source/proximity_list/proximitylist_api.cpp

```diff
@@ -131,9 +131,7 @@ void find(navitia::PbCreator& pb_creator,
                 pb_creator.fill_pb_error(pbnavitia::Error::unable_to_parse,
                                          "Problem while parsing the query:" + parse_error.more);
                 return;
-            } catch (const ptref::ptref_error& pt_error) {
-                pb_creator.fill_pb_error(pbnavitia::Error::bad_filter, "ptref : " + pt_error.more);
-                return;
+            } catch (const std::exception&) {
```

Comment:
I don't know why this is empty ?

Reply:
maybe you should unwind global errors with a generic message ?

Reply:
Here we catch error but continue to search for the rest object types in the query (request). As exception is raised, we are obliged to catch and continue.

Reply:
I got confused at first because I thought we were using `pt_ref` to figure out the proximity list. But `pt_ref` is only here to find out which object are filtered.